pkg/collector/corechecks/containers/docker.go

```diff
@@ -201,6 +201,10 @@ func (d *DockerCheck) Run() error {
 				sender.Gauge("docker.mem.swap", float64(c.Memory.Swap), "", tags)
 			}
 
+			if c.Memory.MemFailCntPresent == true {
```

Comment:
This part should be extending the docker check file to get the Container object's memfailed count metric